src/test/java/com/google/javascript/clutz/DeclarationGeneratorTests.java

```diff
@@ -60,6 +69,15 @@ static File getGoldenFile(final File input) {
     return new File(input.getPath().replaceAll("\\.js$", ".d.ts"));
   }
 
+  static String maybeExternFileName(String testFileName) {
+    String possibleFileName = testFileName.replace(".js",".externs.js");
```

Comment:
done. Obviously, I have gotten too used to clang-format, need to integrate it into my workflow.